src/Utils/util.h

```diff
@@ -73,4 +73,22 @@ namespace Util
 			return std::hash<int>()((int) t);
 		}
 	};
+
+	template<typename N_t = size_t, N_t N>
```

Comment:
howcome you added these if they're not used anywhere

Reply:
The only one I didn't use was the `wchar_t` overload of `StrLen`.